tests/gis_tests/distapp/tests.py

```diff
@@ -323,6 +323,32 @@ def test_geodetic_distance_lookups(self):
             cities = self.get_names(qs)
             self.assertEqual(cities, ['Adelaide', 'Hobart', 'Shellharbour', 'Thirroul'])
 
+    @skipUnlessDBFeature("supports_distances_lookups")
+    def test_distance_lookups_with_expression_rhs(self):
+        qs = SouthTexasCity.objects.filter(
+            point__distance_lte=(self.stx_pnt, F('radius'))
+        ).order_by('name')
+        self.assertEqual(len(qs), 4)
```

Comment:
since we check for the exact elements next, it seems this assertion just makes debugging require a few extra steps to see the items in the list?